src/main/java/dev/meyi/bn/commands/BazaarNotifierCommand.java

```diff
@@ -205,6 +205,14 @@ public void processCommand(ICommandSender ics, String[] args) {
                 player.addChatMessage(help);
               }
               break;
+            case ("show_chat_messages"):
+              BazaarNotifier.config.showChatMessages = !BazaarNotifier.config.showChatMessages;

```

Comment:
```suggestion
              BazaarNotifier.config.showChatMessages ^= true;
```

Let's prefer the bitwise operator here